cypress.json

```diff
@@ -1 +1 @@
-{"video": false}
+{"video": false, "defaultCommandTimeout": 10000 }
```

Comment:
requests has been slow for me maybe cuz I'm testing from China. It doesn't usually takes this long though